Tensile/KernelWriterAssembly.py

```diff
@@ -1999,13 +1999,14 @@ def initKernel(self, kernel, tPA, tPB ):
 
     if kernel["EnableMatrixInstruction"]:
       self.miLatency = kernel["MatrixInstM"] // 2
+      miIssueLatency = 2
       if (self.version == (9,4,0) or self.version == (9,4,1) or self.version == (9,4,2)) and kernel["MatrixInstB"] == 1 and \
          (kernel["EnableF32XdlMathOp"] or \
           kernel["ProblemType"]["DataType"].is8bitFloat() or \
           kernel["ProblemType"]["DataType"].isHalf() or kernel["ProblemType"]["DataType"].isBFloat16() or \
           kernel["ProblemType"]["DataType"].isInt8()):
         self.miLatency //= 2
-      miIssueLatency = 2
+        miIssueLatency //= 2
```

Comment:
Does this affect wait states? Or only used for scheduling?

Reply:
> Does this affect wait states? Or only used for scheduling?

Only for scheduling.
This is used just below to calculate self.miLatencyLeft.
By changning this, self.miLatencyLeft is increased by 1.
This small difference can improve performance a lot.